gradle.properties

```diff
@@ -32,7 +32,7 @@ org.gradle.jvmargs=-XX:MaxMetaspaceSize=512m
 ##########################
 
 GROUP=com.google.android.material
-VERSION_NAME=1.0.0-SNAPSHOT
+VERSION_NAME=1.0.0-beta09
```

Comment:
Please revert this, we tend to do this in a separate commit so that it kicks off a library release workflow